geest/resources/model.json

```diff
@@ -42,7 +42,7 @@
                             "use_point_per_cell": 0,
```

Comment:
You should not edit the mode directly - rather you should modify the generator then regenerate it